modules/installation-osp-modifying-networktype.adoc

```diff
@@ -1,13 +1,12 @@
 // Module included in the following assemblies:
-// * installing/installing_openstack/installing-openstack-installer-user-kuryr.adoc
 //
 //YOU MUST SET AN IFEVAL FOR EACH NEW MODULE
 
 :_mod-docs-content-type: PROCEDURE
 [id="installation-osp-modifying-networktype_{context}"]
 = Modifying the network type
 
-By default, the installation program selects the `OpenShiftSDN` network type. To use Kuryr instead, change the value in the installation configuration file that the program generated.
+By default, the installation program selects the `OpenShiftSDN` network type. To use OVN-Kubernetes instead, change the value in the installation configuration file that the program generated.
```

Comment:
Looks like we've missed this when OVN-Kubernetes became the default, we should fix this like this:

> By default, the installation program selects the `OVN-Kubernetes` network type. To use OpenShift-SDN instead, change the value in the installation configuration file that the program generated.